backend/backend.ts

```diff
@@ -70,6 +73,10 @@ async function createSharedAppObject() {
   };
 }
 
+function createType3MessageFromType2Message(WWWAuthenticate, username, password) {
```

Comment:
```suggestion
export function createType3MessageFromType2Message(WWWAuthenticate, username, password) {
```
And move this into `ntlm.js` of course. (The reason I had this separate is that the `@ewsjs/ntlm-client` exported the functions like this, but I can't call them separately because the return value from `decodeType2Message` doesn't round-trip through JPC.)

Reply:
> the return value from decodeType2Message doesn't round-trip through JPC

Ah, that's the part I was missing. Makes sense.

Why does it not round-trip? Which value does JPC fail to support?